### PR TITLE
Minor improvements to Enterprise SSO section

### PR DESCRIPTION
**Pull request by dfunckt**
Minor changes for consistency and small typo fixes (admittedly pedantic at times) benefiting from fresh perspective.

Change-type: patch

### CODE REVIEW

File: pages/learn/accounts/enterprise-sso.md [hunk 1]
```diff
@@ -9,31 +9,31 @@ __Note:__ This feature is currently available only on [Enterprise plans](https:/
...
+Configuring an Identity Provider as a login method requires a one-time setup process within both balenaCloud and the Identity Provider itself. Refer to our [IdP specific documentation][ms-saml] for detailed instructions on required configurations.
 
 ## Link a SAML Identity Provider
 
-To enable Single Sign-On (SSO) for balenaCloud organizations, you must establish a connection with your external Identity Provider (IdP). BalenaCloud supports all SAML 2.0 Identity Providers, we have provided examples for Microsoft Entra ID (formerly Azure AD) and Google Workspace. This process assumes that you have already configured a [SAML 2.0 IdP and possess an XML certificate][ms-saml] ready for upload.
+To enable Single Sign-On (SSO) for balenaCloud organizations, you must establish a connection with your external Identity Provider. BalenaCloud supports all SAML 2.0 Identity Providers, and we provide examples for Microsoft Entra ID (formerly Azure AD) and Google Workspace. This process assumes that you have already configured a [SAML 2.0 IdP and possess an XML certificate][ms-saml] ready for upload.
```
Comment (dfunckt) [inline pages/learn/accounts/enterprise-sso.md:16]: Would be good to link the IdP config examples from here. A bit lazy to do this now 😁

File: pages/learn/accounts/enterprise-sso.md [hunk 2]
```diff
@@ -47,36 +47,35 @@ __Note__: If you unlink the default team in the Identity Provider configuration,
...
+__Important:__ By activating SAML, you are transferring your personal account to a company account. **This action is non-reversible**. The following changes will occur:
 * **Your API keys will be deleted**
-* **This is a non-reversible action**
 * You will no longer be able to create new API keys
 * Ownership of your fleets and devices will be transferred to your company
 * Your company can revoke your access at any time
 * You will no longer be able to log in using `username` and `password`
 
-You will still be subject to balenaCloud's **terms of service** and **master service agreement**, with the following changes:
+You will still be subject to balenaCloud's **Terms of Service** and **Master Service Agreement**, with the following changes:
```
Comment (dfunckt) [inline pages/learn/accounts/enterprise-sso.md:63]: Same — perhaps worth linking ToS and MAS pages here.

Relatedly, it’s worth considering removing these caveats from here (or if it’s important for the user to understand things to maybe rephrase it in prose), given that a user will not miss it when they try to enable SAML.

File: pages/learn/accounts/enterprise-sso.md [hunk 3]
```diff
@@ -91,7 +90,7 @@ Currently, SAML authentication users cannot create API keys. If you require API
 
 <!-- NOTE: we link to this FAQ in the dashboard -->
 #### How do I delete a SAML account?
-To delete a SAML account, you need to use the `sdk`. This step is only required if you intend to [delete your IdP](#how-do-i-delete-an-identity-provider-in-balenacloud). **Removing the user from your IdP will block their access to balenaCloud**, but their current session will remain active for up to 12 hours after their last login.
+To delete a SAML account, you need to use the balena SDK. This step is only required if you intend to [delete your IdP](#how-do-i-delete-an-identity-provider-in-balenacloud). **Removing the user from your IdP will block their access to balenaCloud**, but their current session will remain active for up to 12 hours after their last login.
```
Comment (dfunckt) [inline pages/learn/accounts/enterprise-sso.md:93]: Same — would be nice to link to SDK docs here.

File: pages/learn/accounts/enterprise-sso.md [hunk 4]
```diff
@@ -9,31 +9,31 @@ __Note:__ This feature is currently available only on [Enterprise plans](https:/
 
 BalenaCloud Enterprise Single Sign-On (SSO) using SAML (Security Assertion Markup Language) allows organizations to manage user access and authentication through their existing Identity Providers (IdP). This integration enables users to use their corporate credentials to log in and access BalenaCloud services. By leveraging SAML, enterprises can simplify the login process, enhance security, streamline user management, and ensure compliance with their internal policies and procedures.
 
-Configuring an Identity Provider (IdP) as a login method requires a one-time setup process within both balenaCloud and the identity provider itself. Refer to our [IdP specific documentation][ms-saml] for detailed instructions on required configurations.
```
Comment (shaunmulligan) [inline pages/learn/accounts/enterprise-sso.md:12]: I think we should leave (IdP) short hand as we later use that instead of writing out the full "Identity Provider". Its common practise to have the short hand in parenthesis the first time you introduce a term and then later in the section only describe it a with the short hand.

**Review by shaunmulligan — COMMENTED** [review]
All the changes look good, the only one I question is removing the instance of short hand for Identity Provider (IdP)

File: pages/learn/accounts/enterprise-sso.md (hunk 4 above)
Comment (dfunckt) [reply pages/learn/accounts/enterprise-sso.md:12]: We already mention the IdP short hand in the paragraph above this one. I think I should add back the shorthand here, but remove it from the first paragraph which is already loaded with terms and shorthands (and undo a commit that I just did which adds the short hand back to a subsequent paragraph).

Comment (vipulgupta2048) [reply pages/learn/accounts/enterprise-sso.md:12]: To me, it doesn't matter as long as it's clear and explicitly mentioned which it was and now is too.

**Comment by vipulgupta2048** [comment]
Additionally, @shaunmulligan did we mention somewhere how people can add a new provider as well which isn't mentioned to give a surety that just because we don't have docs for it that doesn't mean we don't support it. Anywhere we mentioned that?

File: pages/learn/accounts/enterprise-sso.md (hunk 2 above)
Comment (vipulgupta2048) [reply pages/learn/accounts/enterprise-sso.md:63]: Added links

File: pages/learn/accounts/enterprise-sso.md [hunk 5]
```diff
@@ -46,43 +62,47 @@ __Note__: If you unlink the default team in the Identity Provider configuration,
...
 Each SAML Identity Provider (IdP) has its own unique implementation and terminology, which can result in variations in the configuration process. While it is not feasible to provide detailed configuration guidelines for every IdP, we have included example guides for two of the major providers: [Microsoft Entra ID (formerly Azure Active Directory)][ms-saml] and [Google Workspace SAML](/learn/accounts/idp-setup/google-workspace-saml-setup/). These examples are designed to help you understand the necessary steps and how to configure your own provider effectively.
 
+#### Can I use any identity provider with balena’s Enterprise SSO?
+We support any identity provider as long as they are compliant with the SAML 2.0 protocol specification such as Okta, [Microsoft Entra ID][ms-saml] (formerly Azure AD) and [Google Workspace][google-saml]. We have provided examples for using Microsoft Entra ID and Google Workspace in our documentation. To setup your IdP, get the Federation Metadata XML as we have setup for Microsoft/Google IdP's and then follow the [instructions to link that IdP](https://docs.balena.io/learn/accounts/enterprise-sso/#link-a-saml-identity-provider) to balenaCloud's SSO.
```
Comment (dfunckt) [inline pages/learn/accounts/enterprise-sso.md:104]: Good stuff adding this new FAQ entry, and linking the examples. I think, however, that the links should be on the sentence that starts with “We have provided examples …” instead.

**Review by dfunckt — COMMENTED** [review]
Really good improvements @vipulgupta2048, thank you

**Review by vipulgupta2048 — DISMISSED** [review]

File: pages/learn/accounts/enterprise-sso.md (hunk 5 above)
Comment (vipulgupta2048) [reply pages/learn/accounts/enterprise-sso.md]: Made the line better, and leaner instead.

**Review by shaunmulligan — APPROVED** [review]
lgtm
